metrics/metrics/src/main/java/io/helidon/metrics/MetricsSupportProviderImpl.java

```diff
@@ -26,7 +26,7 @@ public class MetricsSupportProviderImpl implements MetricsSupportProvider<Metric
 
     @Override
     public MetricsSupport.Builder builder() {
-        return MetricsSupport.builder();
+        return new MetricsSupport.Builder();
```

Comment:
This should call `MetricsSupport.builder()` and not create a new instance explicitly. I am not sure why this change was made....

Reply:
Changed back.

Reply:
Maybe the method should be changed to package local? Builder should not be instantiated in any other place except for the class that "owns" it.